default.config.yml

```diff
@@ -8,7 +8,8 @@ vagrant_gui: false
 
 # If you need to run multiple instances of Drupal VM, set a unique hostname,
 # machine name, and IP address for each instance.
-vagrant_hostname: drupalvm.dev
+# rfc2606 limits to only use one of .test, .example, .invalid, or .localhost
```

Comment:
Also going to remove this line just because a lot of people (myself included) use something like `local.mysitename.com` for local, and this comment could make that use case confusing.